config/environments/production.rb

```diff
@@ -98,4 +98,6 @@
     compress: true,
     compression_min_size: 524_288
   }
+
+  config.middleware.use Redirector
```

Comment:
It changes middleware from:
```Ruby
use Redirector
use Rack::Attack
use Rack::Deflater
```
to
```RUby
use Rack::Attack
use Rack::Deflater
use Redirector
```
I doubt the changed order would make any difference. `Redirector` just redirects old urls to different websites (`"https://docs.rubygems.org`, `"http://guides.rubygems.org`) which may have existed as a single website at some point.